share/revng/pipelines/analyses-lists.yml

```diff
@@ -0,0 +1,10 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+AnalysesLists:
+  - Name: initial-auto-analysis
+    Analyses:
+      - ImportBinary
+      - AddPrimitiveTypes
+      - DetectABI
```

Comment:
What's the story for the `revng-c` side?
Do we want to support augmenting an existing list with additional analyses or do we simple run a new one from scratch?

IIRC the second one. If so, please make a PR for revng-c too.
Once that's done we can pass things to @mrjackv so he can fix `GraphQL` API and revng-ui.

Reply:
https://rev.ng/gitlab/revng-private/revng-c/-/merge_requests/131/diffs?commit_id=9dadc43af0bc14c2154c80efa9b4c800dc0051fa added here